dtfs-central-api/src/services/utilisation-report-data-validator/utilisation-report-cell-validators/helpers/generate-errors-for-mismatched-facility-values/index.test.ts

```diff
@@ -4,32 +4,35 @@ import { generateErrorsForMismatchedFacilityValues } from '.';
 describe('services/utilisation-report-data-validator/utilisation-report-cell-validators/helpers/generate-errors-for-mismatched-facility-values', () => {
   const field = 'base currency';
   const errorMessage = 'mock error message';
-  const exporterName = 'mock name';
 
   const csvData = [
     {
       'ukef facility id': { value: '20001371', column: 'B', row: 1 },
       'bank facility reference': { value: 'test exporter', column: 'C', row: 1 },
       'base currency': { value: 'GBB', column: 'D', row: 1 },
       'facility utilisation': { value: '2000', column: 'F', row: 1 },
+      exporter: { value: 'test exporter', column: 'C', row: '1' },
     },
     {
       'ukef facility id': { value: '20001371', column: 'B', row: 1 },
-      'bank facility reference': { value: 'test exporter', column: 'C', row: 1 },
+      'bank facility reference': { value: 'test exporter 1', column: 'C', row: 1 },
       'base currency': { value: 'GBP', column: 'D', row: 2 },
       'facility utilisation': { value: '20003', column: 'F', row: 1 },
+      exporter: { value: 'test exporter 1', column: 'C', row: '2' },
     },
     {
       'ukef facility id': { value: '20001371', column: 'B', row: 1 },
-      'bank facility reference': { value: 'test exporter', column: 'C', row: 1 },
+      'bank facility reference': { value: 'test exporter 1', column: 'C', row: 1 },
       'base currency': { value: 'EUR', column: 'D', row: 3 },
       'facility utilisation': { value: '20004', column: 'F', row: 1 },
+      exporter: { value: 'test exporter 1', column: 'C', row: '3' },
     },
     {
       'ukef facility id': { value: '20001372', column: 'B', row: 1 },
-      'bank facility reference': { value: 'test exporter', column: 'C', row: 1 },
+      'bank facility reference': { value: 'test exporter 2', column: 'C', row: 1 },
       'base currency': { value: 'GBP', column: 'D', row: 4 },
       'facility utilisation': { value: '3000', column: 'F', row: 1 },
+      exporter: { value: 'test exporter 2', column: 'C', row: '4' },
```

Comment:
Please could we make this test data more realistic? Each object here represents one row and all properties in one row should have the same row number, otherwise it is quite confusing to follow 😅